jerry-core/parser/js/js-parser.h

```diff
@@ -129,7 +131,18 @@ typedef enum
   PARSER_ERR_REST_PARAMETER_DEFAULT_INITIALIZER,      /**< rest parameter default initializer */
 #endif /* !CONFIG_DISABLE_ES2015_FUNCTION_REST_PARAMETER */
   PARSER_ERR_OBJECT_PROPERTY_REDEFINED,               /**< property of object literal redefined */
-  PARSER_ERR_NON_STRICT_ARG_DEFINITION                /**< non-strict argument definition */
+  PARSER_ERR_NON_STRICT_ARG_DEFINITION,               /**< non-strict argument definition */
+
+#ifndef CONFIG_DISABLE_ES2015_MODULE_SYSTEM
```

Comment:
Move these group before `PARSER_ERR_OBJECT_PROPERTY_REDEFINED`. Therefore the last enum member is not modified.